specs/core/0_beacon-chain.md

```diff
@@ -1654,6 +1654,9 @@ def process_attestation(state: BeaconState, attestation: Attestation) -> None:
     attestation_slot = get_attestation_data_slot(state, data)
     assert attestation_slot + MIN_ATTESTATION_INCLUSION_DELAY <= state.slot <= attestation_slot + SLOTS_PER_EPOCH
 
+    committee = get_crosslink_committee(state, data.target.epoch, data.crosslink.shard)
```

Comment:
Looks good, too bad there is no committee size function. We do not actually need the contents, and the committee size can be derived from total active indices count together with `data.crosslink.shard` (for rounding in committee size).

Reply:
indeed... hrmmm probably not worth adding for sake of freezy-ness